web/src/components/lineage/components/drag-bar/DragBar.tsx

```diff
@@ -4,100 +4,97 @@
 import React from 'react'
 
 import * as Redux from 'redux'
-import { Theme } from '@material-ui/core/styles'
 import { bindActionCreators } from 'redux'
 import { connect } from 'react-redux'
+import { createTheme } from '@mui/material/styles'
 import { setBottomBarHeight } from '../../../../store/actionCreators'
-import Box from '@material-ui/core/Box'
-import classNames from 'classnames'
-import createStyles from '@material-ui/core/styles/createStyles'
-import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles'
-
-const styles = (theme: Theme) =>
-  createStyles({
-    dragBarContainer: {
-      backgroundColor: theme.palette.secondary.main,
-      height: theme.spacing(1),
-      display: 'flex',
-      flexDirection: 'column',
-      alignItems: 'center',
-      justifyContent: 'space-evenly',
-      cursor: 'ns-resize',
-      transition: theme.transitions.create(['background-color']),
-      '&:hover': {
-        backgroundColor: theme.palette.primary.main
-      }
-    },
-    resizing: {
-      backgroundColor: theme.palette.primary.main
-    },
-    slit: {
-      width: theme.spacing(5),
-      height: 1,
-      backgroundColor: theme.palette.common.white
-    }
-  })
+import { useTheme } from '@emotion/react'
+import Box from '@mui/material/Box'
 
 interface DispatchProps {
   setBottomBarHeight: (offset: number) => void
 }
 
-interface DragBarState {
-  isResizing: boolean
-  lastY: number
-}
-
-type DragBarProps = WithStyles<typeof styles> & DispatchProps
+type DragBarProps = DispatchProps
 
 // height of bar / 2
 const MAGIC_OFFSET_BASE = 4
 
-class DragBar extends React.Component<DragBarProps, DragBarState> {
-  constructor(props: DragBarProps) {
-    super(props)
-    this.state = {
-      isResizing: false,
-      lastY: 0
-    }
-  }
+const DragBar: React.FC<DragBarProps> = ({ setBottomBarHeight }) => {
+  const [isResizing, setIsResizing] = React.useState<boolean>(false)
 
-  componentDidMount() {
-    window.addEventListener('mousemove', e => this.handleMousemove(e))
-    window.addEventListener('mouseup', () => this.handleMouseup())
-  }
+  console.log('DragBar render', isResizing)
 
-  componentWillUnmount() {
-    window.removeEventListener('mousemove', this.handleMousemove)
-    window.removeEventListener('mouseup', this.handleMouseup)
-  }
+  // React.useEffect(() => {
+  //   window.addEventListener('mousemove', handleMousemove)
+  //   window.addEventListener('mouseup', handleMouseup)
+
+  //   return () => {
+  //     window.removeEventListener('mousemove', handleMousemove)
+  //     window.removeEventListener('mouseup', handleMouseup)
+  //   }
+  // }, [])
```

Comment:
I think we do not want a comment like this lying around.

Reply:
Done